src/App.module.css

```diff
@@ -1,19 +1,29 @@
-:root {
-    --black: #000;
-}
-html{
+html {
     background-color: #f3f3f3;
-
 }
 .container {
     display: flex;
     flex-direction: column;
-    height: 100%;     /*when contents cannot fill the page and you want to put it in the middle then use height: 100% */
-    /* align-items: center; */
-    /* justify-content: center; */
-    /* display: flex; */
-    /* flex-direction: column;*/
-    /* align-items: center;  */
-    /* justify-content: center; */
-    /* font-size: 1rem; */
+    height: 100%;
+}
+
+@media print {
+    /* a bit of a hack to get the sizes right during printing (assuming
+    this is printed on A4/US Letter paper): 
+    https://stackoverflow.com/a/66593515 */
+    @page {
+        size: A3;
```

Comment:
😆 Hah. This is is like the opposite of what I would expect (setting it to a larger size than what you're printing to fixes the scaling issues 🤪)